policy-controller/k8s/status/src/index.rs

```diff
@@ -43,8 +43,11 @@ pub struct Index {
     claims: Receiver<Arc<Claim>>,
     updates: UnboundedSender<Update>,
 
-    http_routes: HashMap<ResourceId, Vec<ParentReference>>,
+    /// Maps HttpRoute ids to a list of their parent refs, regardless of if
+    /// those parents have accepted the route.
```

Comment:
tiny grammar nit, TIOLI
```suggestion
    /// Maps HttpRoute ids to a list of their parent refs, regardless of
    /// whether those parents have accepted the route.
```